plugins/scaffolder/src/components/fields/VirtualizedListbox.tsx

```diff
@@ -17,34 +17,46 @@
 import React from 'react';
 import { FixedSizeList, ListChildComponentProps } from 'react-window';
 
+type HTMLDivProps = React.HTMLAttributes<HTMLDivElement>;
+
 const renderRow = (props: ListChildComponentProps) => {
   const { data, index, style } = props;
   return React.cloneElement(data[index], { style });
 };
+const OuterElementContext = React.createContext<HTMLDivProps>({});
+const OuterElementType = React.forwardRef<HTMLDivElement, HTMLDivProps>(
+  (props, ref) => {
+    const outerProps = React.useContext(OuterElementContext);
+    return <div ref={ref} {...props} {...outerProps} />;
+  },
+);
 
 export const VirtualizedListbox = React.forwardRef<
   HTMLDivElement,
-  { children?: React.ReactNode }
+  HTMLDivProps
 >((props, ref) => {
-  const itemData = React.Children.toArray(props.children);
+  const { children, ...rest } = props;
+  const itemData = React.Children.toArray(children);
   const itemCount = itemData.length;
 
   const itemSize = 36;
 
-  const itemsToShow = Math.min(10, itemCount);
-  const height = Math.max(itemSize, itemsToShow * itemSize - 0.5 * itemSize);
+  const height = itemSize * (Math.min(10, itemCount) + 0.5);
```

Comment:
How come you dropped the `.max` here? I believe that was in place to ensure that an small box is rendered even if the list is empty.

Reply:
Even if itemcount is 0, it will still render a half element size box, so there will always be a box there. I guess I didnt really need to change these lines for this ticket, it had just bugged me that if there were less than 10 items available, the dropdown would always cut the last item in half so I changed it to add 0.5 of an item rather than subtracting it, which meant you couldnt get a 0 size box any more.

Reply:
Ah yep that works, thank you! :+1: